go-controller/pkg/ovn/secondary_layer2_network_controller.go

```diff
@@ -176,7 +177,9 @@ func (h *secondaryLayer2NetworkControllerEventHandler) UpdateResource(oldObj, ne
 
 			return h.oc.addUpdateLocalNodeEvent(newNode, nodeSyncsParam)
 		} else {
-			return h.oc.addUpdateRemoteNodeEvent(newNode)
+			_, syncZoneIC := h.oc.syncZoneICFailed.Load(newNode.Name)
+			syncZoneIC = syncZoneIC || h.oc.isLocalZoneNode(oldNode)
+			return h.oc.addUpdateRemoteNodeEvent(newNode, syncZoneIC)
```

Comment:
I honestly don't understand this commit. How could this be working up to now ?

Reply:
The zoneIC is used to remote ports to other nodes not to other pods, that's why it was working fine.